Fixed Column.Country (#78)
AnyBlok/AnyBlok: doc/CHANGES.rst

1.0.0
-----

> * Fixed alias. The ``Model.aliased`` method link the registry into alias. The goal is 

Reviewer: ```suggestion
* Fixed alias. The ``Model.aliased`` method now binds the registry to the alias. The goal is 
```
